log.go

```diff
@@ -26,19 +26,35 @@ func (l *logger) GetLevel() loggers.Level {
 	return l.baseLog.GetLevel()
 }
 
-func (l *logger) Debug(ctx context.Context, args ...interface{}) {
+func (l *logger) Debugw(args ...interface{}) {
```

Comment:
dont add this version in the lib, let the user handle it.
Also TODO in lib is not good

Reply:
if we want to log items without context then also i have to pass context.ToDo() or context.Background() everywhere this looks ugly. enforcing user to pass context object even if it is not needed

Reply:
in services we have all our functions accepting contexts, in cli apps it makes sense to skip contexts but not in services.
the amount of call which is contextless is very very less. Just for that changing signature for others doesnt make sense.

Another concern is TODO(), which should be used when you are unclear, and library should not have unclear decision, if there requires, should let it to user

Reply:
folks are migrating library to support context on every function.